README.md

```diff
@@ -75,6 +114,73 @@ MyModel.myMethod = function(cb) {
 });
 ```
 
-See the official LoopBack
-[documentation](https://docs.strongloop.com/display/APIC/Using+current+context)
-for more details.
+### Use current authenticated user in remote methods
+
+In advanced use cases, for example when you want to add custom middleware, you
+have to add the context middleware at the right position in the middleware
+chain (before the middleware that depends on
+`LoopBackContext.getCurrentContext`).
+
+**IMPORTANT: `LoopBackContext.perRequest()` detects the situation when it is
+invoked multiple times on the same request and returns immediately in
+subsequent runs.**
+
+Here’s sample code which uses a middleware function to place the currently
```

Comment:
Here is a snippet using a middleware..
The following is an example of using a middleware...